static/app/components/globalDrawer/components.tsx

```diff
@@ -148,7 +312,84 @@ const DrawerContainer = styled('div')`
 `;
 
 const DrawerSlidePanel = styled(SlideOverPanel)`
-  box-shadow: 0 0 0 1px ${p => p.theme.translucentBorder};
+  box-shadow: 0 0 0 1px ${p => p.theme.dropShadowHeavy};
+  border-left: 1px solid ${p => p.theme.border};
+  position: relative;
+  pointer-events: auto;
+
+  --drawer-width: ${DEFAULT_WIDTH_PERCENT}%;
+  --drawer-min-width: ${MIN_WIDTH_PERCENT}%;
+  --drawer-max-width: ${MAX_WIDTH_PERCENT}%;
+
+  width: clamp(
+    var(--drawer-min-width),
+    var(--drawer-width),
+    var(--drawer-max-width)
+  ) !important;
+
+  &[data-resizing] {
+    /* Hide scrollbars during resize */
+    overflow: hidden !important;
+
+    /* Hide scrollbars in Firefox */
+    scrollbar-width: none;
+
+    /* Hide scrollbars in WebKit browsers */
+    &::-webkit-scrollbar {
+      display: none;
+    }
+
+    /* Apply to all scrollable children */
+    * {
+      overflow: hidden !important;
+      scrollbar-width: none;
+
+      &::-webkit-scrollbar {
+        display: none;
+      }
+    }
```

Comment:
Why do we care about this?

Reply:
This was an ask from design (@vuluongj20).